Resources/views/prepairRecovery.html.twig

```diff
@@ -0,0 +1,45 @@
+{% extends 'base.html.twig' %}
+
+{% block page_title %}
+	{{ 'Easy Backup'|trans }}
+{% endblock %}
+{% block page_subtitle %}
+	{{ 'easy_backup.subtitle'|trans }}
+{% endblock %}
+
+{% block main %}
+
+	<div class="box box-secondary data_table">
+		<div class="box-header with-border">
+			<h3 class="box-title">{{ 'File overrides'|trans }}</h3>
+
+			<div class="box-tools pull-right">
+            	<button class="btn btn-box-tool" data-widget="collapse" data-toggle="tooltip" title="" data-original-title="Collapse">
+        			<i class="fas fa-minus"></i>
+    			</button>
+			</div>
+		</div>
+
+		<div class="box-body no-padding">
+			<table class="table dataTable table-striped table-hover">
+				<thead>
+					<tr>
+						<th scope="col">{{ 'Name'|trans }}</th>
+						<th scope="col">{{ 'New'|trans }}</th>
+					</tr>
+				</thead>
+				<tbody>
+					{% if fileOverwrites is defined %}
+						{% for fileNew in fileOverwrites %}
+							<tr>
+								<td>{{ fileNew }}</td>
```

Comment:
twice the same variable, is that correct?

Reply:
I have to check it. But the prepairRecovery action is not yet used. I planned to build some kind of recovery preview as a first step for any recovery, so that a user can see what files will be overwritten etc. But I don't know if will implement it further or may remove it later on. Yet it is not accessable via any UI.